docs/proposals/cluster-default-components-settings.md

```diff
@@ -0,0 +1,31 @@
+# Cluster default component settings webhook
+
+**Author**: Furkhat @furkhat
+
+**Status**: Draft proposal; prototype in progress.
+
+## Goals
+
+*short description of the topic e.g.*
+Add mutating admission webhook that reads configured configmap with default components settings and sets `.spec.componentsOverride` for new clusters.
```

Comment:
I would add some links for reference:
https://github.com/kubermatic/kubermatic/blob/2a489f4e4de8fb38a5ef2248a5873af9da8843d8/pkg/crd/kubermatic/v1/cluster.go#L307
And mention that this would replace the current solution based on a controller:
https://github.com/kubermatic/kubermatic/blob/master/pkg/controller/seed-controller-manager/clustercomponentdefaulter/clustercomponentdefaulter.go#L110-L141

Reply:
I added the link and mentioned the controller in goals section. Thanks!